src/tests/functional/plugin/shared/include/behavior/infer_request/callback.hpp

```diff
@@ -110,37 +109,6 @@ TEST_P(InferRequestCallbackTests, LegacyCastAndSetuserDataGetUserData) {
     ASSERT_EQ(42, userData);
 }
 
-TEST_P(InferRequestCallbackTests, ReturnResultNotReadyFromWaitInAsyncModeForTooSmallTimeout) {
```

Comment:
Why have you removed this test?

Reply:
In scope of GNA cleaning I also removed `Basic_LSTM_S::GetNetwork()`. Because it's used here (and in `ov_infer_request/callback.hpp`) I move GetNetwork() to API2.0 version and remove the test from API1.0 version (to not support function here too)

Reply:
Because `Basic_LSTM_S::GetNetwork()` is also used in NPU behavior tests, I reverted `basic_lstm.hpp` files, but looks like the function may be eliminated because same behavior was reproduced with default test model (need to prove in CI)

Reply:
`Basic_LSTM_S` dependency is removed from NPU in https://github.com/intel-innersource/applications.ai.vpu-accelerators.vpux-plugin/pull/8343